pytorch_blade/pytorch_blade/torch-mlir/lib/utils/env.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "utils/env.h"
+#include "llvm/Support/raw_ostream.h"
+
+namespace mlir {
+namespace torch {
+namespace utils {
+namespace env {
+std::string AsciiStrToLower(const char* cstr) {
```

Comment:
UT?

Reply:
I just copied these functions from pytorch_blade/common_utils. Had added a comment about this.

Reply:
log utils in pytorch_blade depend on PyTorch log system which will introduce PyTorch as a dependence. Furthermore, PyTorch depends on llvm which will be conflict with that in torch-mlir.